### PR TITLE
Add Zero-value Raster in Highest Point Analysis

### PR DESCRIPTION
**Pull request by gkahiu**
Addresses issue #236. 

The Highest Point Analysis (HPA) algorithm is not zero-based and starts at position 1 meaning that `NoData` pixels will be assigned a value of `1` and the corresponding implementation model pixel values will also be incremented by 1 i.e. starting from `2`.  This has an effect on the styling due to the change in pixel values for the respective implementation models. 

~~@Samweli confirm if this is the expected result otherwise we will need to refactor the code handling the pixel value for implementation models to start from `2` instead of `1`~~.

The PR also includes a global zero-value dataset in WGS84 which is reprojected depending on the CRS of the reference layer during the analysis. This is saved in the base directory, under `zero_value_layers` sub-directory, for future re-use.

The styling pixel editor dialog has been refactored by removing the vertical row numbers as well as the description text to indicate that the ordering of pixel values is relative.

### CODE REVIEW

File: src/cplus_plugin/main.py [hunk 1]
```diff
@@ -504,3 +502,11 @@ def initialize_report_settings():
 
     if settings_manager.get_value(Settings.REPORT_CI_LOGO, None) is None:
         settings_manager.set_value(Settings.REPORT_CI_LOGO, CI_LOGO_PATH)
+
+
+def initialize_zero_value_raster_path():
```
Comment (Samweli) [inline src/cplus_plugin/main.py:507]: We need to include the latest workflow so this initialization can be run each time user installs a new plugin version.

**Review by Samweli — CHANGES_REQUESTED** [review]

File: src/cplus_plugin/data/default/implementation_models.json [hunk 2]
```diff
@@ -284,27 +284,6 @@
             "model_layer": {"color_ramp": "PiYG"}
         }
     },
-    {
```
[the remark this answers is not in the document]
Comment (gkahiu) [reply src/cplus_plugin/data/default/implementation_models.json:287]: This was a duplicate entry.

File: src/cplus_plugin/main.py (hunk 1 above)
Comment (gkahiu) [reply src/cplus_plugin/main.py:510]: This has now been incorporated. In the long term, we might want to consider removing boilerplate code by the caller for checking, removing and updating the settings.

**Comment by gkahiu** [comment]
HPA logic was resolved in subsequent releases.

**Pull request closed without merging**